lib/taste_tester/windows.rb

```diff
@@ -0,0 +1,90 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+# Copyright 2013-present Facebook
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module TasteTester
+  module Windows
+    PATH_TO_WRAPPER_SCRIPT =
+      File.absolute_path(
+        File.join(
+          __FILE__,
+          '\..\..\..\scripts\win_chef_zero_server.ps1',
+        ),
+      )
+
+    def start_win_chef_zero_server
+      script_opts = ''
+      host = 'localhost'
+      port = @state.port
+      ps_opts = '-noprofile -noninteractive'
+      script_opts = '-UseSSL' if @state.ssl
+      cmd = format(
+        'powershell.exe %s -file %s -CZHost %s -CZPort %s %s',
```

Comment:
now that we don't have the crazy pid thing, can't we just run chef-zero directly here? Do we need the wrapper script?